### PR TITLE
[CAPT-1729] Use GOVUK formbuilder for multiple eligibility

### PR DESCRIPTION
**Pull request by asmega**
# Context

- https://dfedigital.atlassian.net/browse/CAPT-1729
- Ultimate goal is to use GovukFormBuilder to simplify forms

# Changes

- Use `GOVUKDesignSystemFormBuilder::FormBuilder` for multiple eligibility form

### CODE REVIEW

File: app/views/additional_payments/claims/_eligibility_confirmed_multiple.html.erb [hunk 1]
```diff
@@ -1,53 +1,26 @@
...
+  <%= f.govuk_submit "Apply now" %>
 <% end %>
 
 <p class="govuk-body">
   For more information about why you are eligible for different additional payments, read about the
-  <%= link_to "early-career payments (opens in new tab)", Policies::EarlyCareerPayments.eligibility_page_url, class: "govuk-link govuk-link--no-visited-state", target: "_blank" %>
+  <%= govuk_link_to "early-career payments (opens in new tab)", Policies::EarlyCareerPayments.eligibility_page_url, no_visited_state: true, target: "_blank" %>
   and
-  <%= link_to "levelling up premium payments (opens in new tab)", Policies::LevellingUpPremiumPayments.eligibility_page_url, class: "govuk-link govuk-link--no-visited-state", target: "_blank" %>.
+  <%= govuk_link_to "levelling up premium payments (opens in new tab)", Policies::LevellingUpPremiumPayments.eligibility_page_url, no_visited_state: true, target: "_blank" %>.
```
Comment (peteryates) [inline app/views/additional_payments/claims/_eligibility_confirmed_multiple.html.erb:25]: Don't mean to butt in just saw this  PR in the feed - you can use the recently-added `new_tab: true` which automatically appends  `(opens in new tab)` and also sets the `rel` and `target` attrs. More info [here](https://govuk-components.netlify.app/helpers/link/#links-that-open-in-a-new-tab).

File: app/views/additional_payments/claims/eligibility_confirmed.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <% if @form.single_choice_only? %>
       <%= render partial: "eligibility_confirmed_single", locals: { policy: @form.policies_eligible_now.first } %>
     <% else %>
-      <%= render partial: "eligibility_confirmed_multiple", locals: { policies: @form.policies_eligible_now_and_sorted } %>
+      <%= render partial: "eligibility_confirmed_multiple" %>
```
[the remark this answers is not in the document]
Comment (asmega) [reply app/views/additional_payments/claims/eligibility_confirmed.html.erb]: good spot, i've now removed it

File: app/views/additional_payments/claims/_eligibility_confirmed_multiple.html.erb [hunk 3]
```diff
@@ -1,53 +1,26 @@
-<%= render("shared/error_summary", instance: @form, errored_field_id_overrides: {
-  "selected_claim_policy": "claim_selected_claim_policy_#{@form.first_eligible_compact_policy_name}"
-}) if @form.errors.any? %>
+<%= form_for @form, url: claim_path(current_journey_routing_name), builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
```
[the remark this answers is not in the document]
Comment (asmega) [reply app/views/additional_payments/claims/_eligibility_confirmed_multiple.html.erb]: yes, I've started doing that on the new forms but didn't think to do that on the existing ones. I've swapped it over here and will do so for any more that i migrate

File: app/views/additional_payments/claims/_eligibility_confirmed_multiple.html.erb (hunk 1 above)
Comment (asmega) [reply app/views/additional_payments/claims/_eligibility_confirmed_multiple.html.erb]: @peteryates no not all, I've made your suggested change here. Thanks again for the great library and making our lives easier!

**Pull request merged**